website/docs/compute/compute.md

```diff
@@ -3,13 +3,15 @@ pagination_prev: null
 ---
 # Suborbital Compute
 
-### The SaaS extensibility platform
+![Suborbital logo](/img/suborbital-logo-wide.svg)
```

Comment:
A bit of a nit, but it looks like this (and the header removal) are the only things that changed in this file. Ideally, other changes like formatting are left out and done in separate PRs. It makes it much easier to track what's actually changing!

Reply:
Will do next time!